openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx

```diff
@@ -421,11 +421,10 @@ export const getGraphDataByTierType = (rawData: TotalEntitiesByTier[]) => {
   };
 };
 
-export const getTeamFilter = (suggestionValues: ListValues = []) => {
-  return (suggestionValues as ListItem[]).map((suggestion: ListItem) => ({
-    label: suggestion.title,
-    value: suggestion.value,
-  }));
+export const getTeamFilter = (suggestionValues: ListValues = []): string[] => {
+  return (suggestionValues as ListItem[]).map(
```

Comment:
Why we are expecting `suggestionValues` to be `ListValues` and then we are casting it to `ListItem[]`?

Reply:
suggestionValues will be ListValues and ListValues can be ListItem[], (string | number)[],
but you are correct it should be handled at page level and not in utility method! i have update the code!